AsyncDisplayKit/ASTableView.mm

```diff
@@ -639,7 +637,7 @@ - (void)handleBatchFetchScrollingToOffset:(CGPoint)targetOffset
 - (void)rangeControllerBeginUpdates:(ASRangeController *)rangeController
 {
   ASDisplayNodeAssertMainThread();
-  LOG(@"--- UITableView beginUpdates");
+  ASLOG(@"--- UITableView beginUpdates");
```

Comment:
I think this is probably a good idea, but I'm worried about not being able to control logging on a per-file basis.  It will get too verbose pretty quickly.